src/main.js

```diff
@@ -33,6 +33,7 @@ const sponsoredTexts = [
   "ได้รับการสนับสนุน", // Thai
   "Sponsorlu", // Turkish
   "Được tài trợ", // Vietnamese
+  "Χορηγούμενη", // Greek
```

Comment:
Can you put them in the sorted order based on Language name?

Reply:
fixed rebasing